### PR TITLE
Connecting to TiDB with VS Code: added instructions on how to install SQLTools MySQL/MariaDB/…

### PR DESCRIPTION
**Pull request by kenudeh**
…TiDB from VS Code extensions tab

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

I added a method of installing the SQLTools MySQL/MariaDB/TiDB extension directly from VS Code's extensions tab. The existing methods do not cover this, despite it being a common way to add extensions among developers.

The new method involves clicking on the extensions tab within VS Code, entering the extension's name, and selecting the extension from the displayed search result. I compressed the instructions into two short sentences.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

https://docs.pingcap.com/tidbcloud/dev-guide-gui-vscode-sqltools#connect-to-tidb

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-gui-vscode-sqltools.md
Comment (dveeden) [inline develop/dev-guide-gui-vscode-sqltools.md:25]: Wouldn't it be better to search for the name of the extension in case some other extension gets published that also matches this search?

image

Comment (dveeden) [reply develop/dev-guide-gui-vscode-sqltools.md:25]: Another option, Ctrl-+P and then enter this command:
```
ext install mtxr.sqltools-driver-mysql
```


image

**Review by Icemap — APPROVED** [review]

**Comment by rpaik** [comment]
@Icemap is anything else needed here for this PR?

**Comment by Icemap** [comment]
> @Icemap is anything else needed here for this PR?

IMO, we need to wait for the author @kenudeh to have some feedback on that comment above. And we need another "LGTM" from the doc team.

File: develop/dev-guide-gui-vscode-sqltools.md
Comment (qiancai) [reply develop/dev-guide-gui-vscode-sqltools.md:25]: Agree, searching with `mtxr.sqltools-driver-mysql` is more efficient. As for "Ctrl-+P", as "Ctrl" varies slight among different operation systems, I think it is OK not to mention that.  Hence, here are my suggested changes. @kenudeh, PTAL

```suggestion
    - In the extensions tab of your VS Code, search `mtxr.sqltools-driver-mysql` to get the **SQLTools MySQL/MariaDB/TiDB**  exertion and click **Install**. 
```

Comment (kenudeh) [reply develop/dev-guide-gui-vscode-sqltools.md:25]: @dveeden, yes, it's a lot better to search using the extension's name. In addition to the remark @qiancai made about your second suggestion, I think using the search feature to install extensions in VS Code is a more common practice.
 @qiancai, thank you for the suggested changes. I sent this PR in haste to beat the deadline during the Docs Dash. I'll commit them now.

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**
